### PR TITLE
Fixes #1

### LINKED ISSUE #1: Todolist
- [x] and
- [x] bitAnd
- [x] or
- [x] bitOr
- [x] not
- [x] compose
- [x] nand
- [x] nor
- [x] nth
- [x] noop
- [x] apply
- [x] pipe
- [x] length
- [x] inc
- [x] dec
- [x] equal
- [x] looseEqual
- [ ] lt
- [ ] gt
- [ ] le
- [ ] ge
- [ ] bind
- [ ] call
- [ ] has
- [ ] converge
- [ ] isString
- [ ] isObject
- [ ] isBoolean
- [ ] isNumber
- [ ] isInt
- [ ] isPlainObject
- [ ] plus
- [ ] minus
- [ ] times (I don’t like this name)
- [ ] by
- [ ] isTrue
- [ ] isFalse
- [ ] isNull
- [ ] isUndefined

### Function
- [x] curry1,2,3,4

### Array functions
- [x] filter(function, array)
- [x] reduce(function, array)
- [x] every(function, array)
- [x] some(function, array)
- [x] reduceRight(function, array)
- [x] reduceFrom(function, initialValue, array)
- [x] reduceRightFrom(function, initialValue, array)
- [ ] forEach(function, array)

### String functions
- [x] join(string, array)
- [x] split(splitStr, string)
- [ ] word
- [ ] csv

### RegExp
- [ ] replace(regEx, string)
- [ ] match(regEx, string)

### PR DESCRIPTION
**Pull request by hemanth**
bitAnd and nor

#1

### CODE REVIEW

**Comment by stoeffel** [comment]
You only changed the name in the `readme` of `and` to `bitAnd` and not the actual file.

I would prefer:

`and` === &&
`bitAnd` === &
`or` === ||
`bitOr` === |

@hemanth what do you think?

**Comment by hemanth** [comment]
@stoeffel My bad, just fetched upstream and merged, had not added bitAnd, now it's in place.

File: src/and.js [hunk 1]
```diff
@@ -1 +1 @@
-export default (a, b) => a && b;
+export default (x, y) => x & y;
```
Comment (stoeffel) [inline src/and.js]: could you revert this change?

Comment (hemanth) [inline src/and.js]: duh

**Comment by hemanth** [comment]
Thanks, was squashing the commits.

**Comment by stoeffel** [comment]
> Thanks, was squashing the commits.

no problem. No need to squash.
